fiatconnect-api.md

```diff
@@ -2213,6 +2214,23 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 }
 ```
 
+####  8.3.2.4. `BankAccount`
+
+`BankAccount` is a Fiat Account schema that tends to come up with a bank account representation we can use regardless of the country. Meanwhile, the most important field there is the `iban` that stands for International Bank Account Number. It respects a specific organization that you can find here [**IBAN on WIKIPEDIA**](https://fr.wikipedia.org/wiki/International_Bank_Account_Number) and [**IBAN**](https://www.pricebank.fr/questions-reponses/questions-frequentes/qu-est-ce-que-le-code-iban.html).
```

Comment:
```suggestion
`BankAccount` is a representation of a bank account that is agnostic to the user's home country. The most important field is the `iban`, which stands for International Bank Account Number. More info on [Wikipedia](https://fr.wikipedia.org/wiki/International_Bank_Account_Number) and [pricebank.fr FAQs page](https://www.pricebank.fr/questions-reponses/questions-frequentes/qu-est-ce-que-le-code-iban.html).
```
rewording for clarity

Reply:
In particular, "tends to" makes it sound like the description only applies part of the time

Reply:
oh also, since the FiatConnect spec is written in English, I think it would be preferable to have English links for more information-- just noticed these are in french

Reply:
OK I will figure out a new link to replace this one. Thanks